noxfile.py

```diff
@@ -87,7 +88,28 @@ def test(session: nox.Session) -> None:
     if _should_install(session):
         session.install("-e", ".[test]")
     session.run(*split("pybabel compile -d src/pydata_sphinx_theme/locale -D sphinx"))
-    session.run("pytest", *session.posargs)
+    session.run("pytest", "-m", "not a11y", *session.posargs)
+
+
+@nox.session()
+def a11y(session: nox.Session) -> None:
+    """Run the accessibility test suite only."""
+    if _should_install(session):
+        session.install("-e", ".[test]")
+        # Install the drivers that Playwright needs to control the browsers.
+        if os.environ.get("CI") or os.environ.get("GITPOD_WORKSPACE_ID"):
+            # CI and other cloud environments are potentially missing system
+            # dependencies, so we tell Playwright to also install the system
+            # dependencies
+            session.run("playwright", "install", "--with-deps")
```

Comment:
I had initially removed this until I had to update the CI tests and realised I duplicated a lot of code trying to use `pytest` directly like in other tests. 
So, for now, I am leaving this here and calling tests through `nox`.

Note: I am unsure if we should keep the `GITPOD_WORKSPACE_ID` bit. I am a Gitpod user, but I wonder if it will confuse the rest of the folks who do not use Gitpod.

Reply:
I hadn't heard of Gitpod, but I wasn't confused :)  I figured "probably some containerized thing" (I've since looked it up).